### ISSUE TITLE
[a11y] [WCAG 2.4.4]: The screen reader does not announce that a link will open in another tab.

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#9917]
OBSERVED RESULTS:
--
When navigating the TestCity CiviForm page with enable screen reader, the link 'civiform-dev@exygy.com' is read as 'civiform-dev@exygy.com Visited link'. However, when pressing 'Enter' on the link, it opens in a new tab, but the screen reader does not announce that the link will open in a new tab beforehand.

USER IMPACT:
--
The Screen reader users may have difficulty understanding the purpose of a link if it’s not clear from the link text or surrounding context, leading to confusion or frustration.

EXPECTED RESULTS:
--
When navigating the TestCity CiviForm page with enable screen reader, the link 'civiform-dev@exygy.com' should be announced with information that it will open in a new tab, such as 'civiform-dev@exygy.com, opens in a new tab'.

STEPS TO REPRODUCE:
--
0. Enable ChromeVox using "CONTROL+ALT+Z".
1. Go to [TestCity CiviForm](https://ns.civiform.dev/programs)
2. Select english language
3. Apply for "Minimal Sample Program"
4. Select the 'Start an application' button
5. Go through application flow and submit 
6. Locate the link 'For technical support: civiform-dev@exygy.com' in the footer of the page
7. Note that the screen reader announces it only as a link, but when you select it, a new tab will open.

SCREENCAST:
--
- Chromebook+ChromeVox:[Evidence](https://drive.google.com/file/d/1SoqDg00Rk64vyK8eP_Ljho8xA2rXArvQ/view?usp=drive_link&resourcekey=0-F0EMtGBc8Q2ZumfU-_iJvQ)
- Windows+NVDA:[Evidence](https://drive.google.com/file/d/1XNPSvuIj3e6S4SH_G8qWodqnMSi5HWta/view?usp=drive_link&resourcekey=0-AkuaIpUdMzt9NGxijlWaNA)
                                

ADDITIONAL INFORMATION:
--
Same behaviour is observed in Not eligible program page: \
Screencast:Chromebook+Chromevox[here]( https://drive.google.com/file/d/1vpcrxbA1rtOSeCh2rYSwl20hVnHXmL7K/view?usp=drive_link )

**Wcag Rule:**

[WCAG 2.4.4](https://www.w3.org/TR/WCAG22/#link-purpose-in-context)

**Enviroment details:**

Machine: Chromebook Version 13.3

Browser: Chrome (132.0.6834.208 (Official Build) (64-bit))

Machine: Windows 11 Enterprise 23H2

Screen reader: ChromeVox, NVDA

Bug Filling Time: 20 mins

**Comment by tallulahkay** [comment]
@dkatzz this issue mentions "Same behaviour is observed in Not eligible program page:" - do you know if there's another issue to fix that? or should I also address that with this issue

**Comment by dkatzz** [comment]
> [@dkatzz](https://github.com/dkatzz) this issue mentions "Same behaviour is observed in Not eligible program page:" - do you know if there's another issue to fix that? or should I also address that with this issue

Good catch! It doesn't look like that was filed as a separate issue, so let's plan to handle that consistent do what we decide to do here. Thank you!

**Comment by dkatzz** [comment]
I had asked the accessibility reviewer about the question around mailto links and this was what they mentioned: `Got it, For mailto links, it's best to describe that the link will open the user's email client, as this depends on their personal configuration. A good aria-label option would be: Send an email to [civiform-dev@exygy.com](mailto:civiform-dev@exygy.com). This will open your email client. This provides clarity without assuming specific browser or app behavior.`

**Comment by melanie-exygy** [comment]
> I had asked the accessibility reviewer about the question around mailto links and this was what they mentioned: `Got it, For mailto links, it's best to describe that the link will open the user's email client, as this depends on their personal configuration. A good aria-label option would be: Send an email to [civiform-dev@exygy.com](mailto:civiform-dev@exygy.com). This will open your email client. This provides clarity without assuming specific browser or app behavior.`

@dkatzz, do you think there's enough of a difference between this suggestion and what we implemented to reopen the issue?  If so, I suggest that we make the issue much lower priority since adding "This will open your email client" is only a nice-to-have.

**Comment by dkatzz** [comment]
> > I had asked the accessibility reviewer about the question around mailto links and this was what they mentioned: `Got it, For mailto links, it's best to describe that the link will open the user's email client, as this depends on their personal configuration. A good aria-label option would be: Send an email to [civiform-dev@exygy.com](mailto:civiform-dev@exygy.com). This will open your email client. This provides clarity without assuming specific browser or app behavior.`
> 
> [@dkatzz](https://github.com/dkatzz), do you think there's enough of a difference between this suggestion and what we implemented to reopen the issue? If so, I suggest that we make the issue much lower priority since adding "This will open your email client" is only a nice-to-have.

I'm looking at the change and realizing we don't have the aria label on the footer element? https://github.com/civiform/civiform/blob/main/server/app/views/applicant/NavigationFragment.html#L337-L356. Am I missing something? I agree it isn't a big change but super quick if we want to do it. I'm happy to do it too if we decide we want to move forward

**Comment by melanie-exygy** [comment]
> I'm looking at the change and realizing we don't have the aria label on the footer element? https://github.com/civiform/civiform/blob/main/server/app/views/applicant/NavigationFragment.html#L337-L356. Am I missing something? I agree it isn't a big change but super quick if we want to do it. I'm happy to do it too if we decide we want to move forward

It's preferable for the label to be the link text (the text between the opening and closing tags of the anchor element).  That's the first place that the screen reader looks for a label.  The nice thing about that is that it keeps the experience the same for sighted users and screen reader users.  That's the ideal.  If that's seeming too long for the visible text, I'd suggest putting an envelope icon (standard icon for email) and making the alt text "This will open your email client".  As a third option, you could make the "This will open your email client" screen-reader only text with the USWDS `sr-only` class.

**Comment by dkatzz** [comment]
Chatted offline with @tallulahkay and @melanie-exygy and decided to keep the solution as is, since it matched what USWDS does and seems compliant with WCAG
